src/modules/lua.c

```diff
@@ -843,6 +843,15 @@ package_manip_path(char *in, const char *find, const char *replace) {
   return npath;
 }
 
+static int MTEV_JIT_OFF(void) {
+  static int enabled = -1;
```

Comment:
enabled is a bad name as this is actually disabling JIT.

Can you change `enabled` to `jit_off`

Reply:
Will do. (Agreed that is bad naming, it was blindly copied from `DEBUG_LOG_ENABLED` in `utils/mtev_log.c`..)